### ISSUE TITLE
3_8_6 AI는 가위바위보를 할 수 있을까? 에서 softmax 함수 누락된건가요?

### ISSUE THREAD

**Issue by Dokkabei97** [needleworm/bhban_ai#2]
p159에서 출력층 활성화 함수는 softmax라고 적혀 있는데

P160에서 소스를 보면 

model = keras.Sequential([
             ...
    keras.layers.Dense(3)
]) 로 되있는데 keras.layers.Dense(3, activation='softmax')로 되어야 하는거 아닌가요?
